src/tickets/tickets.service.ts

```diff
@@ -0,0 +1,95 @@
+import { httpErrors } from '@fastify/sensible'
+import { sql } from 'drizzle-orm'
+import { FastifyInstance, FastifyRequest } from 'fastify'
+import { TCreateTicketSchema } from './schema/create-ticket.schema'
+import { getUserBySessionId } from '../auth/auth.service'
+import { TUpdateTicketSchema } from './schema/update-ticket.schema'
+
+export const getTicketsHandler = async (app: FastifyInstance) => {
+  const response = app.db.query.tickets.findMany({
+    with: { attachments: true },
+  })
+
+  return response
+}
+
+export const getTicketHandler = async (app: FastifyInstance, uuid: string) => {
+  const response = await app.db.execute(sql`
+    SELECT * FROM tickets
```

Comment:
avoid * in selects - you don't need to return everything and in case of adding something - you won't return it automatically and break tests. 

Example : you add ticket_name_embedding column which could be llike 1583 big numbers in an array - which is huge to transport to client.